app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ActiveRecord::Base
+  validates :content, presence: true
+  validates :job_id, presence: true
+
+  belongs_to :job
+
+  def self.order_comments
+    Comment.order("created_at DESC")
```

Comment:
since this is a class method, we don't need to explicitly call `Comment` here - we could just use `self`